jinahub/crafters/ImageNormalizer/normalizer.py

```diff
@@ -45,16 +44,16 @@ def __init__(
         self.img_std = np.array(img_std).reshape((1, 1, 3))
         self.channel_axis = channel_axis
         self.target_channel_axis = target_channel_axis
-        self.logger = JinaLogger(getattr(self.metas, 'name', self.__class__.__name__))
         self.traversal_paths = traversal_paths
+        self.target_dtype = None
 
         # when passed from yaml it is string
         if isinstance(target_dtype, str):
             actual_type = locate(target_dtype)
             if actual_type:
                 self.target_dtype = actual_type
             else:
-                self.logger.error(
+                raise Exception(
```

Comment:
what type of exception do we want to captuer, maybe a bit more explicit?

Reply:
what exception would u raise?

Reply:
Please review again